src/ontology/mondo-edit.obo

```diff
@@ -338910,7 +338913,7 @@ xref: Orphanet:468684 {source="MONDO:equivalentTo"}
 xref: UMLS:C4225191 {source="MONDO:equivalentTo", source="MEDGEN:906792", source="MONDO:MEDGEN"}
 is_a: MONDO:0005501 {source="DC-OMIM:616828", source="DOID:0070267", source="OMIM:616828"} ! congenital disorder of glycosylation type II
 is_a: MONDO:0017749 {source="Orphanet:468684"} ! disorder of multiple glycosylation
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/28178 {source="MONDO:mim2gene_medgen", source="OMIM:616828"} ! CCDC115
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/28178 {source="MONDO:mim2gene_medgen"} ! CCDC115
```

Comment:
can you confirm it is not in the omim file anymore? I see it on their website

Reply:
this also looks like it should have been maintained `relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/28178 {source="MONDO:mim2gene_medgen"} ! CCDC115`